p4-16/spec/P4-16-spec.mdk

```diff
@@ -2815,6 +2831,8 @@ to bit-strings of the same width:
   by `|`.
 - Bitwise "complement" of a single bit-string, denoted by `~`.
 - Bitwise "xor" of two bit-strings of the same width, denoted by `^`.
+- Saturating addition, denoted by `|+|`.
```

Comment:
Another possibility is `[+]`, which suggests confinement to a range.

Reply:
I recommend using with `|+|`. `[]` are too overloaded.